felix/bpf-gpl/conntrack.h

```diff
@@ -788,7 +813,13 @@ static CALI_BPF_INLINE struct calico_ct_result calico_ct_v4_lookup(struct cali_t
 		result.ifindex_fwd = dst_to_src->ifindex;
 	}
 
-	CALI_CT_DEBUG("result: %d\n", result.rc);
+	if (syn) {
+		CALI_CT_DEBUG("packet is SYN\n");
```

Comment:
What about SYN-ACK packets, seems like they'd get the new treatment too?

Reply:
no

`bool syn = tcp_header && tcp_header->syn && !tcp_header->ack;`